### DISCUSSION TITLE
how to export the result in vtk.format?

### DISCUSSION THREAD

**Discussion by junjunjanna** [e0404/matRad#671] [Q&A]
hello everyone~

I made a plan for prostate cancer, and I want to export the CT and dose in VTK.format.
However, I don't have access to enter full screen so I cannot find a place to select the export folder. Could anyone tell me how to get the export data?

<img width="370" alt="Screenshot 2023-11-04 at 18 00 18" src="https://github.com/e0404/matRad/assets/114455710/982957d7-d096-41bf-aac6-b2ef6ef6d6bf">
<img width="182" alt="Screenshot 2023-11-04 at 18 00 24" src="https://github.com/e0404/matRad/assets/114455710/80958621-dee7-4a3a-8397-9ef6a745d277">

**Comment by wahln** [comment]
Dear @junjunjanna,
yeah, this is an unfortunate bug - depending on the screen resolution, the window might not be shown correctly. Are you working with the standalone or with the Matlab code?

There's a "trick" to get access to the browse dialog in the export (it works in windows, not sure about mac). You can "tab" into the invisible dialog - basically, if you cylcle through the options in the dialog with the tab key on your keyboard, the "browse" button will be selected two tabs after the cancel button is selected (the lower right button that is only partly visible in your screenshot).

THis will be fixed in a future version. If you are working with the matRad code within Matlab, I can point you towards a different solution.
